README.md

```diff
@@ -13,7 +13,7 @@ written in plain language, they can be read by anyone on your team. Because they
 read by anyone, you can use them to help improve communication, collaboration and trust on
 your team.
 
-Cucumber.js is the JavaScript implementation of Cucumber and runs on the [maintained Node.js versions](https://github.com/nodejs/Release) and *modern* web browsers.
+Cucumber.js is the JavaScript implementation of Cucumber and runs on the [maintained Node.js versions](https://github.com/nodejs/Release).
 
 ## Try it now
```

Comment:
Should this section go away?

Reply:
I think it does a job in terms of explaining what this project is in context of the wider cucumber universe.

Reply:
What do you think?

Reply:
I think I wasn't clear, I meant the "try it now" section. The previous one is definitely good!